### PR TITLE
TokensCounter package added

### PR DESCRIPTION
**Pull request by yaroslavyaroslav**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My [package](https://github.com/yaroslavyaroslav/SublimeTokenCounter) is simply counting selected text as token, which is useful for folks that are working with llm. Is uses phantoms to present the calculations.

There's no package alike that, because there were lack of dependencies to perform this operation. Now they are available in PC.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by yaroslavyaroslav** [comment]
Looks like tests failing not by my fault.

File: repository/t.json [hunk 1]
```diff
@@ -2897,6 +2897,19 @@
 				}
 			]
 		},
+		{
+			"name": "TokensCounter",
+			"details": "https://github.com/yaroslavyaroslav/SublimeTokenCounter",
+			"issues": "https://github.com/yaroslavyaroslav/SublimeTokenCounter/issues",
```
Comment (braver) [inline repository/t.json:2903]: That's the default URL for issues so no need to add it here, PackageControl will find that automatically.

```suggestion
```

**Review by braver — CHANGES_REQUESTED** [review]
Took me a while to realize what kind of tokens 😅 Package looks good though. Just one minor remark about your entry here.

**Comment by yaroslavyaroslav** [comment]
@braver could you please check it once again, as I applied fixes that have been requested.

**Pull request merged**